indico/modules/categories/util.py

```diff
@@ -78,7 +78,7 @@ def get_min_year(category_id=None):
     :return: A `datetime` Object.
     """
     category_filter = Event.category_chain_overlaps(category_id) if category_id else True
-    return db.session.query(db.func.min(Event.created_dt)).filter(~Event.is_deleted, category_filter).scalar()
+    return db.session.query(db.func.min(Event.start_dt)).filter(~Event.is_deleted, category_filter).scalar()
```

Comment:
what's the reason for changing this?

Reply:
We probably don't want the max `created_dt`. We group events by their `start_dt` and then we also want to get the max `start_dt` not `created_dt`. Unless there is a reason for this?

Reply:
I think I got it. LGTM!

Reply:
I was using `created_dt` as it is only truly useful for the lower bound